docs/root/configuration/other_features/dlb.rst

```diff
@@ -0,0 +1,42 @@
+.. _config_connection_balance_dlb:
+
+DLB Connection Balancer
```

Comment:
```suggestion
DLB connection balancer
```

@daixiang0 here you use `DLB` whereas elsewhere you use `Dlb` - i think we want to make this consistent

Reply:
Let me change it to `Dlb`.